### PR TITLE
Provide some instructions on using TAR ball

### PR DESCRIPTION
**Pull request by timhallinflux**
Highlighting how to setup the datastore for Chronograf when using TAR ball.

### CODE REVIEW

File: content/chronograf/v1.3/introduction/getting-started.md [hunk 1]
```diff
@@ -210,6 +210,27 @@ sudo dpkg -i chronograf_1.3.1.0_amd64.deb
 sudo systemctl start chronograf
 ```
 
+##### Using the TAR packages
+If you choose to use the TAR packages, as opposed to the operating system package management tools (such as yum, wget, etc.), there are a couple of things that you should be aware of.
+
+First, when you start-up Chronograf, we recommend that you specify a location for the underlying datastore as a default location is not specified.  If you do NOT specify a location for the underlying datastore, a datastore named chronograf-v1.db will be created in the directory from which you start Chronograf.
```
Comment (lukevmorris) [inline content/chronograf/v1.3/introduction/getting-started.md]: I believe we're using semantic linefeeds throughout the docs repo, so we should break these out into multiple lines:

```
First, when you start-up Chronograf, ...
If you do not specify a location ...

Second, specifying ...
```

etc

File: content/chronograf/v1.3/introduction/getting-started.md [hunk 2]
```diff
@@ -210,6 +210,27 @@ sudo dpkg -i chronograf_1.3.1.0_amd64.deb
...
+
+Second, specifying the location of the underlying datastore is important because as you take advantage of future release of Chronograf (maintenance releases or feature bearing), you will want to refer to this underlying datastore to preserve your dashboards and datasource configuration.  If you simply un-TAR the downloaded package and restart Chronograf, you will reinitalize the local datastore -- and it will appear as though all of your previous configurations and dashboards have "disappeared."  The package management tools provided by the operating system deal with the "version" changes in the underlying software, but when usint TAR, the version number is embedded in the directory stucture and you end up with multiple side-by-side versions of the software.
+
```
Comment (lukevmorris) [inline content/chronograf/v1.3/introduction/getting-started.md]: E.g., with semantic linefeeds, it'd be easier for me to target the typo in the paragraph above:

" ... underlying software, but when **using** TAR, the version number ... "

File: content/chronograf/v1.3/introduction/getting-started.md [hunk 3]
```diff
@@ -210,6 +210,27 @@ sudo dpkg -i chronograf_1.3.1.0_amd64.deb
...
+The recommended way to address this issue when using the TAR package is to do the following:
+
+In your home directory, create a sub-directory called .chronograf
+```
+mkdir ~/.chronograf
+```
+
+When starting up Chronograf use the -b option to specify the location of this directory for the local datastore:
```
Comment (lukevmorris) [inline content/chronograf/v1.3/introduction/getting-started.md]: I would surround -b in ticks (`):

`-b`

File: content/chronograf/v1.3/introduction/getting-started.md [hunk 4]
```diff
@@ -210,6 +210,27 @@ sudo dpkg -i chronograf_1.3.1.0_amd64.deb
...
+```
+./chrongraf-<version>/usr/bin/chronograf -b ~/.chronograf/chronograf-v1.db
+```
+
+The next time you un-TAR a new version of Chronograf, you can simply change the <version> portion of the command string and re-point to your existing chronograf-v1.db datastore.
```
Comment (lukevmorris) [inline content/chronograf/v1.3/introduction/getting-started.md]: I might also surround `chronograf-v1.db` here, and in the instances above

File: content/chronograf/v1.3/introduction/getting-started.md (hunk 1 above)
Comment (timhallinflux) [inline content/chronograf/v1.3/introduction/getting-started.md]: done.

File: content/chronograf/v1.3/introduction/getting-started.md (hunk 2 above)
Comment (timhallinflux) [inline content/chronograf/v1.3/introduction/getting-started.md]: fixed

File: content/chronograf/v1.3/introduction/getting-started.md (hunk 4 above)
Comment (timhallinflux) [inline content/chronograf/v1.3/introduction/getting-started.md]: done

File: content/chronograf/v1.3/introduction/getting-started.md (hunk 3 above)
Comment (timhallinflux) [inline content/chronograf/v1.3/introduction/getting-started.md]: done

File: content/chronograf/v1.3/introduction/getting-started.md (hunk 2 above)
Comment (timhallinflux) [inline content/chronograf/v1.3/introduction/getting-started.md]: fixed

**Comment by lukevmorris** [comment]
Nice - thanks @timhallinflux!

**Comment by rkuchan** [comment]
This looks great, @timhallinflux! I was wondering if you'd be OK if I move your content from the getting started guide to the installation page. I like having a minimal getting started guide and providing more in-depth instructions on the installation page.

**Pull request merged**